examples/FeatureDiscovery.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Button from '../src/Button';
+import FeatureDiscovery from '../src/FeatureDiscovery';
+
+export default
+<div>
+  <Button large floating fabClickOnly fab='horizontal' icon='edit' waves='light' className='red' id='menu'>
+    <Button floating icon='insert_chart' className='red'/>
+    <Button floating icon='format_quote' className='yellow darken-1'/>
+    <Button floating icon='publish' className='green'/>
+    <Button floating icon='attach_file' className='blue'/>
+  </Button>
+
+  <FeatureDiscovery open className='red white-text' target='menu'>
```

Comment:
Do note though that I've added a `mounted` bool. That-is so we make sure the button
that is set as the `target` is rendered first, and thus `$('#' + target)` succeeds.

The component will therefore be rendered and open once `componentDidMount` is called.
So in a sense yes; it will be open as soon as it is rendered.